components/event-bus/cmd/event-publish-service/application/application.go

```diff
@@ -3,6 +3,7 @@ package application
 import (
 	"net/http"
 
+	messagingv1alpha1Client "github.com/knative/eventing/pkg/client/clientset/versioned/typed/messaging/v1alpha1"
```

Comment:
Capital letters in package aliases overlap with the variable names domain, better avoid them.
https://rakyll.org/style-packages/#lowercase-only